client/src/theme.ts

```diff
@@ -1,38 +1,62 @@
 import { createMuiTheme } from "@material-ui/core/styles";
 import { Fab } from "@material-ui/core";
 
+const PRIMARY_COLOR = "#66BE64";
+const SECONDARY_COLOR = "#909090";
+const SUCCESS_COLOR = "#00E676";
+const ERROR_COLOR = "#F44336";
+
+const WHITE_TEXT_COLOR = "white";
+const WHITE_BACKGROUND_COLOR = "#FFFFFF";
+
 // https://material-ui.com/customization/theming/#createmuitheme-options-args-theme
 // Customize our app-specific theme
 const AppTheme = createMuiTheme({
   palette: {
     primary: {
-      main: "#66BE64",
-      contrastText: "white",
+      main: PRIMARY_COLOR,
+      contrastText: WHITE_TEXT_COLOR,
     },
     secondary: {
-      main: "#909090",
-      contrastText: "white",
+      main: SECONDARY_COLOR,
+      contrastText: WHITE_TEXT_COLOR,
     },
   },
   typography: {
     body2: {
       fontWeight: 800,
     },
     subtitle2: {
-      color: "#909090",
+      color: SECONDARY_COLOR,
     },
   },
 });
 
+// Moving 'success' and 'error' into primary/secondary to allow
+// usage with majority of components (typescript + @material-ui constrain)
 export const ErrorTheme = createMuiTheme({
   palette: {
     primary: {
-      main: "#FFFFFF",
-      contrastText: "#00E676",
+      main: SUCCESS_COLOR,
+      contrastText: WHITE_TEXT_COLOR,
+    },
+    secondary: {
+      main: ERROR_COLOR,
+      contrastText: WHITE_TEXT_COLOR,
+    },
+  },
+});
+
+// Inverse of ErrorTheme with text-color and backgrouund color inverted
```

Comment:
is this for dark mode?

Reply:
no, this is 'default' mode
![Screenshot 2020-07-01 at 15 35 11](https://user-images.githubusercontent.com/5269791/86216357-7d5b6000-bbb0-11ea-9354-b26baaf84abc.png)
currently, only 'primary' and 'secondary' are valid options for the `color` prop in most material-ui components, but perhaps at least we need the 'success' and 'error' ones too, I'm just moving those colors into the 'primary' and 'secondary' fields and wrapping a different `<MuiThemeProvider theme={} />` to allow us to configure these consistent color schemes and use them in components

Reply:
gotcha! So whats the difference between LightErrorTheme & ErrorTheme in terms of usage?

Reply:
LightErrorTheme has the colors flipped around, where the text color is 'success' or 'error' and main color's 'white' instead, eh, perhaps `InvertedErrorTheme` is better?